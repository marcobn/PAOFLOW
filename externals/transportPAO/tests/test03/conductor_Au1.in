
  &INPUT_CONDUCTOR
    postfix='_Au1_PROJ'
    work_dir = './SCRATCH'
    dimC = 9
    calculation_type = 'bulk'
    ne = 1000  
    emin = -10.0
    emax =   5.0 
    datafile_C = "./SCRATCH/gold1.save/atomic_proj.xml"
    transport_dir = 3
    delta = 0.0001
  /

<HAMILTONIAN_DATA>

   <H00_C  cols="1-9" rows="1-9" />
   <H_CR   cols="1-9" rows="1-9" />

</HAMILTONIAN_DATA>

