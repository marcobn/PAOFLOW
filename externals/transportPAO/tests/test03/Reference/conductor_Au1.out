
  ======================================================================
              =                                            =
              =     *** WanT *** Wannier Transport Code    =
              =        (www.wannier-transport.org)         =
              =      Ultra Soft Pseudopotential Implem.    =
              =                                            =
  ======================================================================


  Program <conductor>  v. 2.5.0alpha2  starts ...
  Date 17Apr2014 at 18:34:29 

     Number of MPI processes:       8



          BUILT :    Thu 17 Apr 2014 06:13:20 PM CEST
           HOST :    x86_64-unknown-linux-gnu
           ARCH :    x86_64
             CC :    icc
            CPP :    cpp
            F90 :    mpif90
            F77 :    ifort
         DFLAGS :    -D__INTEL -D__FFTW -D__MPI -D__PARA
      BLAS LIBS :    -L/home/ferretti/want/install -lmkl_em64t_nothread
    LAPACK LIBS :    
       FFT LIBS :    
      MASS LIBS :    



  ======================================================================
  =  Conductor Init                                                    =
  ======================================================================

  ./SCRATCH/gold1.save/atomic_proj.xml file fmt: atmproj
   ATMPROJ conversion to be done using: 
     atmproj_nbnd :     10
     atmproj_thr  :      0.900000
     atmproj_sh   :     10.000000

  ./SCRATCH/gold1.save/atomic_proj.xml converted from ATMPROJ to internal fmt

  ======================================================================
  =  INPUT Summary                                                     =
  ======================================================================

  <INPUT>
          Calculation Type :     bulk
                    prefix :     
                   postfix :     _Au1_PROJ
                  work_dir :     ./SCRATCH
               L-lead dim. :         9
            conductor dim. :         9
               R-lead dim. :         9
       Conductance Formula :     landauer
                  Carriers :     electrons
       Transport Direction :         3
          Have Correlation :     FALSE
              Write k-data :     FALSE
            Write sgm lead :     FALSE
                Write gf C :     FALSE
           Max iter number :       200
                    nprint :        20

        Conductor datafile :     ./SCRATCH/gold1.save/atomic_proj.xml.ham
       leads are identical :     TRUE 
         ovp orthogonaliz. :     FALSE
  </INPUT>


  <ENERGY_GRID>
                 Dimension :       1000
                 Buffering :          1
                Min Energy :      -10.00000
                Max Energy :        5.00000
               Energy Step :        0.01502
                     Delta :        0.00010
             Smearing Type :     lorentzian
             Smearing grid :      20000
             Smearing gmax :       25.00000
                   Shift_L :        0.00000
                   Shift_C :        0.00000
                   Shift_R :        0.00000
                Shift_corr :        0.00000
  </ENERGY_GRID>


  <K-POINTS>
       nkpts_par =    1
       nrtot_par =    1
        use_symm = TRUE

       Parallel kpoints grid:        nk = (  1  1  1 )   s = (  0  0  0 )
       k (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000

       Parallel R vector grid:       nr = (  1  1  1 )
       R (   1) =    (   0.00000  0.00000  0.00000 ),   weight =   1.0000
  </K-POINTS>


  <PARALLELISM>
       Paralellization over frequencies
       # of processes:     8
  </PARALLELISM>


 <MEMORY_USAGE>
                smearing:          0.480 MB
                 kpoints:          0.000 MB
        hamiltonian data:          0.020 MB
        correlation data:          0.000 MB
               workspace:          0.000 MB

     Total alloc. Memory:          0.501 MB
 </MEMORY_USAGE>


  ======================================================================
  =  Frequency Loop                                                    =
  ======================================================================

  Computing E(     1 ) =    -10.00000 eV
  T matrix converged after avg. # of iterations      9.500

                              Total time spent up to now :     0.18 secs

  Computing E(    20 ) =     -9.71471 eV
  T matrix converged after avg. # of iterations      9.500

                              Total time spent up to now :     0.20 secs

  Computing E(    40 ) =     -9.41441 eV
  T matrix converged after avg. # of iterations      9.500

                              Total time spent up to now :     0.22 secs

  Computing E(    60 ) =     -9.11411 eV
  T matrix converged after avg. # of iterations      9.000

                              Total time spent up to now :     0.23 secs

  Computing E(    80 ) =     -8.81381 eV
  T matrix converged after avg. # of iterations      9.000

                              Total time spent up to now :     0.25 secs

  Computing E(   100 ) =     -8.51351 eV
  T matrix converged after avg. # of iterations      9.000

                              Total time spent up to now :     0.27 secs

  Computing E(   120 ) =     -8.21321 eV
  T matrix converged after avg. # of iterations      9.000

                              Total time spent up to now :     0.29 secs

  Computing E(   125 ) =     -8.13814 eV
  T matrix converged after avg. # of iterations      9.000

                              Total time spent up to now :     0.29 secs


  ======================================================================
  =  Writing data                                                      =
  ======================================================================


  conductance written on file:    cond_Au1_PROJ.dat

  doscond written on file:    doscond_Au1_PROJ.dat

  ======================================================================

   <global routines>
             clock number :    15

                conductor :     0.00s CPU 

           datafiles_init :     0.12s CPU
      atmproj_to_internal :     0.08s CPU
             compute_rham :     0.00s CPU (      18 calls,   0.000 s avg)
            smearing_init :     0.02s CPU
                   cft_1z :     0.01s CPU (       2 calls,   0.005 s avg)
         hamiltonian_init :     0.00s CPU
              read_matrix :     0.00s CPU (       2 calls,   0.002 s avg)
              fourier_par :     0.00s CPU (       4 calls,   0.000 s avg)
             do_conductor :     0.14s CPU
        hamiltonian_setup :     0.00s CPU (     125 calls,   0.000 s avg)
            transfer_mtrx :     0.09s CPU (     125 calls,   0.001 s avg)
                    green :     0.01s CPU (     250 calls,   0.000 s avg)
              gzero_maker :     0.00s CPU (     375 calls,   0.000 s avg)
            transmittance :     0.00s CPU (     125 calls,   0.000 s avg)


